src/main/java/org/openmrs/addonindex/domain/artifactory/CoreVersionsSummary.java

```diff
@@ -0,0 +1,102 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v. 2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+ * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+ *
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
+
+package org.openmrs.addonindex.domain.artifactory;
+
+import java.util.List;
+
+
+public class CoreVersionsSummary {
```

Comment:
I think this class should be called ArtifactoryFolderInfo. And it should have a line of javadoc saying what it's for.

Further, I would prefer to remove any fields that we're not using. (It's also confusing to see created/lastModified/lastUpdated as Strings instead of some kind of date type.)